### PR TITLE
Update README for release

### PR DESCRIPTION
**Pull request by jsharpe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,27 @@ in future changes as these older versions are not tested.
...
+- Added repository rules for downloading prebuilt versions of cmake and ninja
+  rather than relying on system installed tools.
+
+- Added native ninja build rule
+
+- Now builds under the bazel sandbox rather than in /tmp
+
+- Tidied up the structure of the examples directory
+
+- Deprecated the old rules `install_ws_dependency`
```
Comment (UebelAndre) [inline README.md:38]:
```suggestion
- Deprecated the old rules `install_ws_dependency` and `cc_configure_make`
```

File: README.md [hunk 2]
```diff
@@ -20,6 +20,27 @@ in future changes as these older versions are not tested.
...
+
+- Deprecate the old `cc_configure_make` rule
```
Comment (UebelAndre) [inline README.md:40]: No need for these to be two different line items.

**Review by UebelAndre — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -20,6 +20,27 @@ in future changes as these older versions are not tested.
...
+
+- Added native ninja build rule
+
+- Now builds under the bazel sandbox rather than in /tmp
+
+- Tidied up the structure of the examples directory
+
+- Deprecated the old rules `install_ws_dependency` and `cc_configure_make`
+
+- Deprecate the old `cc_configure_make` rule
```
Comment (jsharpe) [inline README.md:40]:
```suggestion
```

**Comment by UebelAndre** [comment]
@googlebot I fixed it.

**Review by UebelAndre — APPROVED** [review]

**Pull request merged**